src/pymonzo/monzo_api.py

```diff
@@ -300,6 +302,30 @@ def balance(self, account_id=None):
 
         return MonzoBalance(data=response.json())
 
+    def pots(self, refresh=False):
+        """
+        Returns a list of pots owned by the currently authorised user.
+        Official docs:
```

Comment:
100% a nitpick, but would you mind adding empty lines before `Official docs:` and `:param refresh:` so it's in the same format as other docstrings? Thanks!